listenbrainz/webserver/testing.py

```diff
@@ -22,7 +22,7 @@ class ServerTestCase(unittest.TestCase):
     @classmethod
     def create_app(cls):
         app = create_web_app(debug=False)
-        app.config['TESTING'] = True
```

Comment:
No need to make any change here I think, all of the code except the one for setting up databases in manage.py and for setting connect strings in init_app should not touch PYTHON_TESTS_RUNNING and just use TESTING.